logbook-core/src/main/java/org/zalando/logbook/DefaultLogbook.java

```diff
@@ -35,6 +39,7 @@
     @Override
     public Optional<Correlator> write(final RawHttpRequest rawHttpRequest) throws IOException {
         if (writer.isActive(rawHttpRequest) && predicate.test(rawHttpRequest)) {
+            final Instant start = Instant.now(clock);
```

Comment:
I was thinking about duration as about correlation duration which in fact it is.
Thus I would make correlation start time a property of correlation itself, i.e. refactor `String correlationId` into value object with uuid and start time.
I think it also makes sense to name it correlationTime or correlationDuration to explicitly note that it is not roundtrip time.